src/devices/cpu/mb86235/mb86235.cpp

```diff
@@ -1,22 +1,17 @@
 // license:BSD-3-Clause
-// copyright-holders:Angelo Salese, ElSemi, Ville Linde
+// copyright-holders:Angelo Salese, ElSemi, Ville Linde, Matthew Daniels
 /********************************************************************************
  *
  * MB86235 "TGPx4" (c) Fujitsu
  *
- * Written by Angelo Salese & ElSemi
+ * Written by Angelo Salese, ElSemi & Matthew Daniels
```

Comment:
It's suggested to just use `copyright-holders`, no need to repeat in header (this is a pre-license remnant, where we didn't have SPDX identifiers)